keras_cv/models/models_test.py

```diff
@@ -156,6 +158,18 @@ def _test_model_can_be_used_as_backbone(self, app, last_dim, args):
         model = keras.Model(inputs=inputs, outputs=[backbone_output])
         model.compile()
 
+    def _test_model_serialization(self, app, _, args, save_format, filename):
```

Comment:
Part of me wants to include compiling the model and making sure e.g the optimizer gets serialized but I suppose this is probably redundant with upstream Keras tests about serialization. wdyt?